packages/types/src/defaults.ts

```diff
@@ -0,0 +1,7 @@
+// eslint-disable-next-line @typescript-eslint/no-namespace -- Ellegance reasons/
```

Comment:
We should probably disabled this lint if we do like to use it a lot

Reply:
It will be now.